projects/xlnt/build.sh

```diff
@@ -28,9 +28,11 @@ CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) cmake --build .
 CXX="clang++"
 CXXFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
 
-$CXX $CXXFLAGS -I/xlnt/include -I/xlnt/third-party/libstudxml -O2 -o load_fuzzer.o -c ../load_fuzzer.cc
+$CXX $CXXFLAGS -I/xlnt/include -I/xlnt/third-party/libstudxml -O2 -o load_fuzzer.o -c ../load.cc
```

Comment:
Maybe we can compile and link within same command?

Reply:
done